data/mods/Aftershock/items/inactiverobot.json

```diff
@@ -219,5 +219,119 @@
       "place_randomly": true,
       "skills": [ "electronics", "computer" ]
     }
+  },
+  {
+    "id": "bot_laserturret_x-ray",
+    "type": "TOOL",
+    "name": { "str": "inactive laser turret" },
+    "description": "This is an inactive laser turret.  Using this item involves turning it on and placing it on the ground, where it will attach itself.  If reprogrammed and rewired successfully the turret will identify you as a friendly, and attack all enemies with its replaceable laser cannons.  It requires ammunition tubes to fire.",
+    "weight": "40750 g",
+    "volume": "30 L",
+    "price": 600000,
+    "price_postapoc": 12000,
+    "to_hit": -3,
+    "bashing": 8,
+    "material": [ "steel", "plastic" ],
+    "symbol": ";",
+    "color": "white",
+    "use_action": {
+      "type": "place_monster",
+      "monster_id": "mon_laserturret_x-ray",
+      "difficulty": 6,
+      "moves": 100,
+      "skills": [ "electronics", "computer" ]
+    }
+  },
+  {
+    "id": "bot_utilibot_beehive",
+    "type": "TOOL",
+    "copy-from": "bot_utilibot_base",
+    "name": "inactive bee-bot",
+    "description": "A salvaged utility robot converted into an ambulatory bee hive that periodically removes and delivers segments of fresh honey comb.  It protects the insect colony with a mechanical crossbow mounted to its chassis.  Activate this item, with wooden bolts in your inventory, to load and deploy the robot.",
```

Comment:
This sounds very ridiculous, I'm not sure if we really want it?

Reply:
So I've mostly been using this to experiment with what can be done with biosignatures, and if I can get them to work properly I was going to rewrite this as a turret/stationary robot with no weapon that works as a beehive for harvest-able honey.  But as is no this probably shouldn't  stay.  And if you think the beehive is a deadend I'll drop it entirely.

Reply:
Yeah If you turn it into some sort of "ai monitored beehive" it seems percectly fine for me.